doctrine/orm-toolbox.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "registrar": [
+        {
+            "language": "php",
+            "provider": "doctrine.models",
+            "signature": [
+                "Doctrine\\ORM\\EntityManagerInterface::copy",
```

Comment:
That's not the same.

The entry in `signature` responds to: `$reference = $entityManager->getReference('<CARET>', 5);`; providing autocompletion for all `doctrine.models`.
Afterwards, the entry in `signatures` responds to `$reference-><CARET>`; providing autocompletion on `$reference` based on the type (index 1 by default) into `getReference` call.

But you are right, the `copy` makes no sense here, as you pass in the object to copy, not a string.